src/common/mfu_io.c

```diff
@@ -120,6 +388,21 @@ int mfu_lstat(const char* path, struct stat* buf)
     return rc;
 }
 
+/* calls lstat, and retries a few times if we get EIO or EINTR */
+int mfu_file_lstat(const char* path, struct stat* buf, mfu_file_t* mfu_file)
+{
+    if (mfu_file->type == POSIX) {
+        int rc = mfu_lstat(path, buf);
+        return rc;
+    } else if (mfu_file->type == DAOS) {
+        int rc = daos_stat(path, buf, mfu_file);
+        return rc;
+    } else {
+        MFU_ABORT(-1, "File type not known: %s errno=%d (%s)",
+                  path, errno, strerror(errno));
```

Comment:
errno won't be well defined for this kind of error, but let's report the type value instead
```suggestion
        MFU_ABORT(-1, "File type not known: %s type=%d",
                  path, mfu_file->type);
```
Let's update any other spots like this for unknown mfu_file type values